python/ray/tune/syncer.py

```diff
@@ -60,8 +53,8 @@ def set_sync_periods(sync_config: "SyncConfig"):
             "'TUNE_CLOUD_SYNC_S' is deprecated. Set "
             "`cloud_sync_period` via tune.SyncConfig instead.")
         CLOUD_SYNC_PERIOD = env_integer(key="TUNE_CLOUD_SYNC_S", default=300)
-    NODE_SYNC_PERIOD = int(sync_config.node_sync_period)
-    CLOUD_SYNC_PERIOD = int(sync_config.cloud_sync_period)
+    NODE_SYNC_PERIOD = int(sync_config.sync_period)
+    CLOUD_SYNC_PERIOD = int(sync_config.sync_period)
```

Comment:
why take aways the feature?

Reply:
We consolidate the syncers, i.e. there is either a node syncer (when no cloud syncing) or there is a cloud syncer (and no between-node syncing) - so we never have both at the same time (anymore).

The cloud syncer is created in the trial runner for experiment checkpointing. If no cloud-checkpointing is used, it is no-op.
The node syncer is created in the syncer callback. If cloud-checkpointing is used, it is no-op.

Trial checkpointing (previously using the durable trainable) directly interfaces with the sync clients (not the syncer classes), so does not care about this feature. However for reference, it returns a function-based sync client (e.g. an s3 command) if a string is passed or a default cloud syncer if an upload dir is passed